src/views/site.html

```diff
@@ -1,46 +1,59 @@
 <main id="content" role="main">
 
-  {{> common/betaBanner}}
+  {{> common/betaBanner }}
 
   <div class="grid-row">
     <div class="column-two-thirds">
-      <h1 class="heading-large" id="site-heading">What's the site name?</h1>
-      <p>
-        The site is the area you want the permit to cover.
-      </p>
+
+      {{> common/errorSummary }}
+
+      <h1 id="site-heading" class="heading-large">{{ pageHeading }}</h1>
+
+      <p>The site is the area you want the permit to cover.</p>
 
       <form method="POST" action="site">
-        <div class="form-group">
+
+        <div class="form-group {{#getFormGroupClass errors.siteName}}{{/getFormGroupClass}}">
           <label class="form-label" for="siteName">
             <span id="site-name-label">Site name</span>
             <span class="form-hint">If it doesn’t already have a name, enter the name you’ll use</span>
           </label>
-          <input class="form-control form-control-char-60" id="siteName" name="siteName" type="text" value="">
+
+          {{> common/fieldError fieldId='site-name-error' message=errors.siteName }}
+
+          <input class="form-control form-control-char-60" id="siteName" name="siteName" type="text" value="{{formValues.siteName}}">
+        </div>
+
+        <div class="form-group {{#getFormGroupClass errors.anotherName}}{{/getFormGroupClass}}">
+          <label class="form-label" for="anotherName">
+            Another name
+            <span class="form-hint">If it doesn’t already have a name, enter the name you’ll use</span>
+          </label>
+
+          {{> common/fieldError message=errors.anotherName }}
+
+          <input class="form-control form-control-char-60" id="anotherName" name="anotherName" type="text" value="{{formValues.anotherName}}">
         </div>
 
         <div class="form-group">
           <button id="site-submit" type="submit" class="button">Continue</button>
         </div>
+
       </form>
+
     </div>
   </div>
 
   <p>
     Example partial with CSS styling from the assets/sass folder:
   </p>
+
   <p>
     {{> example}}
   </p>
 
-  {{{ message }}}
-
-  <br>
-  <br>
-
-  Site entered: {{ siteName }}
-  {{#if errors}}
-    <h2 id="site-error">ERROR: {{errors.message}}</h2>
-  {{/if}}
+  <br />
+  <br />
```

Comment:
Think these should be missing the closing /

Reply:
My editor was adding the closing / automatically. I have updated the setting and removed these characters as I do not believe there is any requirement to confirm to "xhtml1-strict.dtd" DOCTYPE